vignettes/classes-objects.Rmd

```diff
@@ -257,6 +257,7 @@ Range <- new_class("Range",
       class = class_double,
       getter = function(self) self@end - self@start,
       setter = function(self, value) {
+        if (length(value) == 0) return(self)
```

Comment:
@t-kalinowski it probably makes sense for this to be an explicit `is.null()` test, right?

Reply:
I don't think so. The default value passed in by the constructor will be `double(0)`. If we want the default to be `NULL`, we'd have to change the property class to `NULL | class_double`, or implement a custom constructor.